core/app/models/spree/app_dependencies.rb

```diff
@@ -51,7 +52,8 @@ def set_default_services
       @checkout_get_shipping_rates_service = 'Spree::Checkout::GetShippingRates'
 
       # sorter
-      @order_sorter = 'Spree::Orders::Sort'
+      @collection_sorter = 'Spree::BaseSorter'
+      @order_sorter = 'Spree::BaseSorter'
```

Comment:
are we using this `@order_sorter`  ?  in controllers ?

Reply:
@przemosk we're not , but still people can you use it in their code as it was available in previous versions